src/Package.php

```diff
@@ -134,6 +134,8 @@ private function deduceTrackingCode(): Package
             $carrier_code = Carrier::CODE_DHL;
         } elseif (preg_match('/^[1-9]{4}[0-9]{4}[0-9]{4}$/', $tracking_code, $matches)) {
             $carrier_code = Carrier::CODE_FEDEX;
+        } elseif (preg_match('/^[1-9]{4}[0-9]{4}[0-9]{4}[0-9]{3}$/', $tracking_code, $matches)) {
```

Comment:
@simoheinonen I think this will break support for 12 digit FedEx codes? Does the end pattern need to be `[0-9]{0,3}` instead?